packages/shared/src/lib/core/nfts/interfaces/persisted-nft-metadata.interface.ts

```diff
@@ -0,0 +1,14 @@
+import { NftStandard } from '../enums'
+import { INftInstance } from '../interfaces'
+import { NftMetadata } from '../types'
+
+export interface IPersistedNftMetadata {
+    standard: NftStandard
+    metadata: NftMetadata
+    instances?: INftInstance[]
+    contentType?: string
+    contentLength?: string
+    responseCode?: number
+    downloadUrl?: string
+    error?: { message: string }
```

Comment:
in erc721 this is per instance 
where in irc27 there is only 1 nft

Reply:
We should just store an object for each instance and not nest them by collection